src/item.cpp

```diff
@@ -209,6 +209,29 @@ item::item( const itype *type, time_point turn, solitary_tag )
 item::item( const itype_id &id, time_point turn, solitary_tag tag )
     : item( find_type( id ), turn, tag ) {}
 
+item::item( const recipe *rec, long qty, t_item_vector items )
```

Comment:
Should be passed by reference here. Also, I'd get rid of the alias - it adds no value apart from obscuring the real type.

Reply:
I totally meant to get rid of that alias in the code cleanup commit, guess I forgot.